__tests__/app.test.js

```diff
@@ -252,3 +252,27 @@ describe('/api/articles/:article_id/comments', () => {
       );
   });
 });
+
+describe('/api/articles/:article_id', () => {
```

Comment:
We need some further testing for our patch, this should also be in a separate describe block to the gets. 
Status 200, updates single article object (decrease votes)
Status 400, invalid ID, e.g. string of "not-an-id"
Status 404, non existent ID, e.g. 0 or 9999
Status 400, incorrect body, e.g. inc_votes property is not a number
(Optional) Status 200, if inc_votes is missing, should not update article and return original single article object (this may be included as a 400 but that could make for a brittle test suite)